src/components/table/helpers/mixin-items.js

```diff
@@ -52,6 +76,21 @@ export default {
         /* istanbul ignore next */
         this.localItems = []
       }
+    },
+    // Watch for changes on computedItems and update the v-model
+    computedItems(newVal, oldVal) {
+      this.$emit('input', newVal)
```

Comment:
Don't we need a `looseEqual()` check here too?
Otherwise we should remove the unused `oldVal` argument.

Reply:
`computedItems` always returns a new array reference when it changes

Reply:
So yeah, we could remove the unused argument var.

Reply:
Removed the unused argument